packages/compass-collection/src/components/collection-tab.tsx

```diff
@@ -149,11 +149,9 @@ function useCollectionTabs(props: CollectionMetadata) {
           </WithErrorBoundary>
         ),
         title: (
-          <WithErrorBoundary name={name} type="header">
```

Comment:
why the removed error boundary?

Reply:
So it's [this issue](https://mongodb.slack.com/archives/CFFA5BS79/p1755613123694299?thread_ts=1755608008.740289&cid=CFFA5BS79) caused by the fact that @leafygreen-ui/lib@15 seems like the one that seems to be working for all our dependendecies more or less, but it does cause something weird in leafygreen tabs component specifically: the title that we pass there wrapped in a boundary starts to fail with this error (I'm guessing they are doing something with this value passed that triggers the constructor accidentally). Before tabs were installing their own version, but doing this very targeted override on top of other overrides is a pain in the butt, so I'm just getting around the issue here by removing the boundary from the rendering tree.

I'll leave a todo